doc/source/f2py/common_session.dat

```diff
@@ -1,27 +1,30 @@
 >>> import common
->>> print common.data.__doc__
-i - 'i'-scalar
-x - 'i'-array(4)
-a - 'f'-array(2,3)
+>>> print(common.data.__doc__)
+'i'-scalar
```

Comment:
Why the names of common block items are removed? These are needed as the user wouldn't know how to access the items.

Reply:
This is actual output from the command. Maybe a bug?

Reply:
yes, I think so. Could you create a new issue about this bug?

Reply:
Done, here #15308 . Thanks!